app/src/main/java/com/example/myapplicationwork/adapers/ActorListAdapter.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.myapplicationwork.adapers
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import com.example.myapplicationwork.R
+import com.example.myapplicationwork.modelsClass.Actor
+import com.example.myapplicationwork.modelsClass.Movie
+
+class ActorListAdapter(
+        private var contextData: List<Actor>,
+     //   private val clickListener: OnRecyclerItemClicked
+        //): RecyclerView.Adapter<MoviesListAdapter.ViewHolderMovies>() {
+): RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
if this adapter uses a single view holder type
let's when you declare your adapter class use this type as generic